src/Compatibility/Core/src/Windows/Forms.cs

```diff
@@ -119,11 +119,11 @@ public static void Init(IActivationState state, InitializationOptions? options =
 			{
 				MainWindow = mainWindow;
 
-				if (mainWindow is WindowsBasePage windowsPage)
-				{
-					windowsPage.LoadApplication(windowsPage.CreateApplication());
-					windowsPage.Activate();
-				}
+				//if (mainWindow is WindowsBasePage windowsPage)
```

Comment:
I've commented out WindowsBasePage because it's not a type that we need for compatibility or MAUI